pyaction/PYACTION_GCONPROD_INSERT_KW.DATA

```diff
@@ -0,0 +1,658 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2015-2024 Equinor ASA
+
+-- This model is built from the base case for model2. This case has standard wells, 3 producers and 2 injectors
+-- Well OP03  has modified completions compared to the base case model
+-- The water oil contact of equilibrium region 2 has been moved to get a stronger well water cut development
+--
+-- The model tests the use of the PYACTION and GCONPROD keywords to control gas production.
+-- This run is a depletion case, with no pressure support from the producers or injectors.
+-- The same modifications using ACTIONX keywords are tested with actionx/ACTIONX_GCONPROD.DATA.
```

Comment:
Can we rephrase this and similar sentences please. Maybe along the lines:

"This model performs the modifications done with ACTIONX keywords in model actionx/ACTIONX_GCONPROD.DATA, but using PYACTION instead of ACTIONX."